packages/cli/bin/console

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env php
```

Comment:
We should rename the console name as this as available later as `vendor/bin/console` which is not distinctable between all the other commands. Perhaps something like `t3docs-theme`?